dev_requirements.txt

```diff
@@ -1,8 +1,8 @@
 # install latest changes in dbt-core + dbt-postgres
 # TODO: how to switch from HEAD to x.y.latest branches after minor releases?
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-postgres&subdirectory=plugins/postgres
+git+https://github.com/dbt-labs/dbt-core.git@dbeatty/utils-lift-shift#egg=dbt-core&subdirectory=core
+git+https://github.com/dbt-labs/dbt-core.git@dbeatty/utils-lift-shift#egg=dbt-tests-adapter&subdirectory=tests/adapter
+git+https://github.com/dbt-labs/dbt-core.git@dbeatty/utils-lift-shift#egg=dbt-postgres&subdirectory=plugins/postgres
```

Comment:
just need to remove the branch identifiers, rerun tests, then we should be hot to trot

Reply:
Hot to trot 🔥